tool/example.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:butterfly/butterfly.dart';
+
+class FooWidget extends StatelessWidget {
+  @override
+  Node build() {
+    var local = false;
+    var bar = 2;
+    var items = [1, 2, 3].map((x) => text("${x}")).toList();
+
+    return new MyComponent(foo: bar)([
```

Comment:
Not important right now, but I wonder if we could generate the code in a way that doesn't shift line numbers.